docs/download_files.md

```diff
@@ -92,31 +92,50 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
 
 ## Custom datasets
 
-<!--TODO: Fill in this section with information about creating and downloading custom datasets-->
+To download data for individual samples, multiple projects, or any combination of individual samples and projects, use the `Add to Dataset` button.
+The `My Dataset` button can then be used to view and download the custom dataset as a single zip file. 
 
-## Portal-wide downloads
+<!--TODO: Confirm that these are the correct names that will be used as the folder names for data format-->
+Each zip file will be named with a unique dataset ID, the chosen data format (either `SCE` or `AnnData`), and the date you accessed the data on the ScPCA Portal.
```

Comment:
What about a cart with _spatial-only_? Based on #381, it looks like it would still say SCE in the data format part of the file name even if no SCEs are downloaded, so I wanted to confirm this. Might be another question for @dvenprasad, too.

Reply:
I think the answer to this is yes, since the data "format" for spatial is SCE. So for naming it would still be `{dataset-id} _ {data-format} _ {access-date}` and then there would just be the one folder with `{project_accession}_spatial`.